frontend/src/pages/Mesh/layouts/MeshColaLayout.ts

```diff
@@ -37,7 +37,9 @@ class MeshLayout extends ColaLayout implements Layout {
 
   private childLayouts: BaseLayout[] = [];
 
-  protected initializeLayout(): void {}
+  protected initializeLayout(): void {
+    console.log('init mesh layout');
```

Comment:
Is this comment needed?

Reply:
Removed.  And currently this whole layout needs work and is not being used/exposed.